test/erlcloud_ddb_streams_tests.erl

```diff
@@ -587,3 +588,48 @@ list_streams_output_tests(_) ->
                        table_name = <<"Forum">>}]}})
         ],
     output_tests(?_f(erlcloud_ddb_streams:list_streams()), Tests).
+
+
+undynamize_ddb_streams_record_output_tests(_) ->
+    DecodedResponse = jsx:decode(
+        <<"
+        {
+            \"awsRegion\": \"us-west-2\",
+            \"dynamodb\": {
+                \"ApproximateCreationDateTime\": 1551727994,
+                \"Keys\": {
+                    \"ForumName\": {\"S\": \"DynamoDB\"},
+                    \"Subject\": {\"S\": \"DynamoDB Thread 3\"}
+                },
+                \"SequenceNumber\": \"300000000000000499659\",
+                \"SizeBytes\": 41,
+                \"StreamViewType\": \"KEYS_ONLY\"
+            },
+            \"eventID\": \"e2fd9c34eff2d779b297b26f5fef4206\",
+            \"eventName\": \"INSERT\",
+            \"eventSource\": \"aws:dynamodb\",
+            \"eventVersion\": \"1.0\"
+        }">>),
```

Comment:
I think you want to put `[{return_maps, false}]` explicitly here... for compatibility with newer jsx versions.